adapters/rubicon.go

```diff
@@ -70,16 +69,63 @@ type rubiconPubExt struct {
 }
 
 type rubiconBannerExtRP struct {
-	SizeID int    `json:"size_id,omitempty"`
-	MIME   string `json:"mime"`
+	SizeID 		int    `json:"size_id,omitempty"`
+	AltSizeIDs 	[]int `json:"alt_size_ids,omitempty"`
```

Comment:
Does this imply that the Rubicon OpenRTB endpoint is expecting an `alt_size_ids` array?